### DISCUSSION TITLE
VSCode: cmd-click to open resources in /public

### DISCUSSION THREAD

**Discussion by IanVS** [withastro/language-tools#691] [Feature requests]
### Background & Motivation

It would be great to be able to cmd+click on absolute paths to resources in /public from within astro components.  So, for example, clicking on the path in `<img src="/images/hero.svg"/>` would open up `/public/images/hero.svg`.  

### Possible solutions

Incorporate into VSCode extension.

### Help make it happen!

- [ ] I am willing to submit a PR to implement this change.
- [ ] I am willing to submit a PR to implement this change, but would need some guidance.
- [X] I am not willing to submit a PR to implement this change.

**Comment by aFuzzyBear** [comment]
I believe this would the reference to the documentation for this particular request,
https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition
